src/do-not-disturb.js

```diff
@@ -0,0 +1,58 @@
+// Always false if the platform doesn't support it.
+const LINUX_SETTING_COMMAND = "gsettings get org.gnome.desktop.notifications show-banners";
+
+const { promisify } = require('util');
+const childProcess = require('child_process');
+const exec = promisify(childProcess.exec);
+
+let doNotDisturbMode = false;
+
+function init() {
+    if (process.platform === "linux") {
+        return initForLinux();
+    }
```

Comment:
I'm really sad that can't do the same for windows or other linux platforms. A more complete implementation would be great (or a comment saying why we don't support them)

Reply:
The intent is for others to go and fill in the gaps, rather than expecting me to iterate over all platforms for this feature, given it's custom for each DM :/